GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,10 +8,163 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var episodes = GOTEpisode.allEpisodes
+    
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
```

Comment:
Same here. By not having a proper instance you don't have access to how many seasons you have. By using this instance you can refactor this code:
     return "Season \(section + 1)"